### PR TITLE
Update to elixir 3.3.1, removed unused elixir.json format bower.json

### PR DESCRIPTION
**Pull request by joecohens**
@jbrooksuk This enables browsersync for us :)

### CODE REVIEW

File: elixir.json [hunk 1]
```diff
@@ -1,6 +0,0 @@
-{
-    "bowerDir": "vendor/bower_components",
```
Comment (jbrooksuk) [inline elixir.json]: Do we not need this file?

Comment (joecohens) [inline elixir.json]: We are not using it anymore, I think that key doesn't even exists on elixir config a lot time ago lol

Comment (jbrooksuk) [inline elixir.json]: Oh, cool :)

**Comment by GrahamCampbell** [comment]
@joecohens Could you get this sorted on StyleCI/StyleCI too please?

**Comment by joecohens** [comment]
@GrahamCampbell Sure!

**Pull request merged**